src/main/java/address/browser/BrowserManager.java

```diff
@@ -58,7 +59,7 @@
     };
 
     public BrowserManager(ObservableList<ReadOnlyViewablePerson> filteredPersons) {
-        initializeBrowser();
+        initializeJxBrowserEnvironment();
```

Comment:
Calling BrowserManager.init() before the browserManager instance is quite odd. I'm calling the init after the browserManager instance is created at mainController.

Yes, I will add in the conditional.